File: SoloLearn Codes/Day of the Week/DayOfTheWeek.java
import java.util.*;
import java.text.*;

public class DayOfTheWeek
{
	public static void main(String[] args) throws ParseException {
		String s = new Scanner(System.in).nextLine();
		Date date = new SimpleDateFormat(Character.isLetter(s.charAt(0)) ? "MMMM dd, yyyy" : "MM/dd/yyyy").parse(s);
		System.out.println(new SimpleDateFormat("EEEE").format(date));
	}
}

File: SoloLearn Codes/Day of the Week/day_of_the_week.swift
import Foundation

let s = readLine()!
var dtFmtG = DateFormatter()
let dtFmtP = DateFormatter()
dtFmtP.dateFormat = "EEEE"
let ch = Array(s.lowercased())[0] as Character
dtFmtG.dateFormat = ch >= "a" && ch <= "z" ? "MMMM dd, yyyy" : "MM/dd/yyyy"
let date = dtFmtG.date(from: s)
print(dtFmtP.string(from: date!))